packages/protocol/contracts/interfaces/IMoxieBondingCurveV2.sol

```diff
@@ -40,7 +38,6 @@ interface IMoxieBondingCurveV2 {
         address _orderReferrer
     ) external returns (uint256 shares_);
 
-
```

Comment:
revert these please

Reply:
this is a formatter mismatch, can you try from yours once ? 
I manually tried but still not matching